spark/src/test/scala/geotrellis/spark/summary/polygonal/FeatureCollectionSpec.scala

```diff
@@ -36,14 +36,14 @@ class FeatureCollectionSpec extends FunSpec
       val polygon = Polygon( (2, 2), (4, 6), (6, 2), (2, 2) )
 
       val featureCollection = Seq(lowerExtent, middleExtent, upperExtent).map { e => Feature(e.toPolygon, e.area) }
-      val result = featureCollection.polygonalSummary(polygon, 0.0)(
-        PolygonalSummaryHandler({ feature: Feature[Polygon, Double] => feature.data })
-                           ({ (polygon, feature) => polygon.intersection(feature.geom).asMultiPolygon.map(_.area).getOrElse(0.0) })
-                           ({ (v1, v2) => v1 + v2 })
-      )
-      val expected = polygon.area - 0.5
-
-      result should be (expected)
+//      val result = featureCollection.polygonalSummary(polygon, 0.0)(
```

Comment:
Is this test worth keeping? It implements a custom PolygonalSummaryHandler that summarizes across vector geometries instead of raster ones. I think in the new API, this would be handled by a separate RDD method and isn't something worth supporting at this time unless we know of a valid use case that's actively used.

Reply:
I don't think this is worth keeping and I'm not sure its really part of this API. The difficult part of polygonal summary is that it is over raster. This potentially can be 1) spatial join on intersection 2) map over all intersections 3) reduce. Anything on top of that potentially obscures the meaning and makes the operation less useful. The problem being is that we don't have a great spatial join operation, but we can focus our attention on that.